File: src/main/java/com/eric/okpo_java/lab2/Task1.java
package com.eric.okpo_java.lab2;

import java.util.Scanner;

public class Task1 {

    public static void main(String[] args) {
        int x, y;
        var input = new Scanner(System.in);
        System.out.println("Введите целое число х");
        x = input.nextInt();
        if (x <= 5) y = x + 5;
        else y = 2 * x;
        System.out.println(y);
    }
}

File: src/main/java/com/eric/okpo_java/lab4/Task1.kt
package com.eric.okpo_java.lab4

class Task1 {

    fun main () {
        var S = 0

        for (i in 0..50) {
            if (i % 3 == 0) {
                S += i
            }
        }

        println(S)
    }
}